tests/Aspire.Microsoft.Data.SqlClient.Tests/ConformanceTests.cs

```diff
@@ -1,22 +1,29 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Components.Common.Tests;
 using Aspire.Components.ConformanceTests;
 using Microsoft.Data.SqlClient;
+using Microsoft.DotNet.RemoteExecutor;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Xunit;
 
 namespace Aspire.Microsoft.Data.SqlClient.Tests;
 
-public class ConformanceTests : ConformanceTests<SqlConnection, MicrosoftDataSqlClientSettings>
+public class ConformanceTests : ConformanceTests<SqlConnection, MicrosoftDataSqlClientSettings>, IClassFixture<SqlServerContainerFixture>
 {
+    private readonly SqlServerContainerFixture _containerFixture;
+    protected override bool CanConnectToServer => RequiresDockerTheoryAttribute.IsSupported;
+    private string ConnectionString => RequiresDockerTheoryAttribute.IsSupported
+                                        ? _containerFixture.GetConnectionString()
+                                        : "Data Source=fake;Database=master";
     protected override ServiceLifetime ServiceLifetime => ServiceLifetime.Scoped;
 
     // https://github.com/open-telemetry/opentelemetry-dotnet/blob/031ed48714e16ba4a5b099b6e14647994a0b9c1b/src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs#L31
-    protected override string ActivitySourceName => "OpenTelemetry.Instrumentation.SqlClient";
+    protected override string ActivitySourceName => "Aspire.Microsoft.Data.SqlClient";
```

Comment:
This shouldn't be updated. It needs https://github.com/dotnet/aspire/pull/3118 which will fix this.